common/main/si.xml

```diff
@@ -9589,12 +9589,12 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 	</typographicNames>
 	<personNames>
 		<nameOrderLocales order="givenFirst">und</nameOrderLocales>
-		<nameOrderLocales order="surnameFirst">ko si te vi yue zh</nameOrderLocales>
+		<nameOrderLocales order="surnameFirst">ko si ta te vi yue zh</nameOrderLocales>
 		<foreignSpaceReplacement xml:space="preserve"> </foreignSpaceReplacement>
 		<initialPattern type="initial">{0}.</initialPattern>
 		<initialPattern type="initialSequence">{0} {1}</initialPattern>
 		<personName order="givenFirst" length="long" usage="referring" formality="formal">
-			<namePattern>{given} {given2} {surname} {surname2} {suffix}</namePattern>
+			<namePattern>{prefix} {given} {given2} {surname} {surname2} {suffix}</namePattern>
```

Comment:
This change to add `{prefix}` should be down at lines 9650-9652 in `<personName order="surnameFirst" length="long" usage="referring" formality="formal">`

Reply:
I just retracted this change, and added the {prefix} further down